.github/workflows/ci.yml

```diff
@@ -0,0 +1,12 @@
+name: CI
+on:
+  push:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: olafurpg/setup-scala@v2
+      - run: ./scalafmt --test
```

Comment:
It's think it's nice to separate the formatting checks from tests because it's useful to see which one failed when reviewing a PR

Reply:
ok